src/confluent_kafka/admin/__init__.py

```diff
@@ -277,19 +277,19 @@ def _make_acls_result(f, futmap):
                 fut.set_exception(e)
 
     @staticmethod
-    def _make_user_scram_credentials_result(f, futmap):
+    def _make_futmap_result(f, futmap):
```

Comment:
Can we use latest created `_make_futmap_result_list` function instead?

Reply:
They're different in one the future returns a `dict` and in `_make_futmap_result_list` it returns a `list`